plinth/frontpage.py

```diff
@@ -28,12 +28,15 @@ def get_shortcuts():
 
 
 def add_shortcut(id, label, url, icon, details=None, configure_url=None,
-                 login_required=False):
+                 login_required=False, icon_url=None):
```

Comment:
Since icon_url is no longer a URL, I suggest rename it to something like icon_name (or just icon if we remove the old one).